doc/modules/tree.rst

```diff
@@ -486,15 +486,25 @@ Regression criteria
 -------------------
 
 If the target is a continuous value, then for node :math:`m`,
-representing a region :math:`R_m` with :math:`N_m` observations, a common
-criterion to minimise is the Mean Squared Error
+representing a region :math:`R_m` with :math:`N_m` observations, common
+criteria to minimise are
+
+Mean Squared Error:
 
 .. math::
 
     c_m = \frac{1}{N_m} \sum_{i \in N_m} y_i
 
     H(X_m) = \frac{1}{N_m} \sum_{i \in N_m} (y_i - c_m)^2
 
+Mean Absolute Error:
+
+.. math::
+
+    c_m = \frac{1}{N_m} \sum_{i \in N_m} y_i
+
```

Comment:
I think no need to write `c_m` again, as it  already mention above. Better way is to place `c_m` before `Mean Squared Error` and `Mean Absolute Error` once instead of writing in both section individually.